vignettes/geologic-map.Rmd

```diff
@@ -100,95 +100,74 @@ library(sf)
 
 First, we can use the package `rnaturalearth` to get the outlines of the USA, Canada and Mexico to use as a background. 
 
-```{r load countries, eval = TRUE}
-# Load background information
+```{r load-countries, eval = TRUE}
+# Load country outlines
 n_a <- ne_states(country = "united states of america", returnclass = "sf")
```

Comment:
Is there any reason why this variable isn't called `us`, for consistency? And do we need to download Mexico if we don't then use it?

Reply:
Will change the US designation!